vendor-extra/JatsContentBundle/src/EventListener/ContentHeaderListener.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\EventListener;
+
+use FluentDOM\DOM\Element;
+use Libero\ContentPageBundle\Event\CreateContentPageEvent;
+use Libero\ViewsBundle\Views\ViewConverter;
+
+final class ContentHeaderListener
+{
+    private $converter;
+
+    public function __construct(ViewConverter $converter)
+    {
+        $this->converter = $converter;
+    }
+
+    public function onCreatePage(CreateContentPageEvent $event) : void
+    {
+        $xpath = $event->getItem()->xpath();
+
+        $front = $xpath->firstOf('/libero:item/jats:article/jats:front');
+
+        if (!$front instanceof Element) {
+            return;
+        }
+
+        $title = $xpath->firstOf('jats:article-meta/jats:title-group/jats:article-title', $front);
+
+        if ($title instanceof Element) {
+            $event->setTitle((string) $title);
```

Comment:
in the current model, the last listener wins as `setTitle()` calls always override an already present title; should this method raise an exception if called multiple times?

Reply:
This case should check it's not already set. There might be reasons to decorate it.